etc/st2packgen/templates/action_template.yaml.jinja

```diff
@@ -21,11 +21,11 @@ parameters:
   {%- for parameter in paramsreq %}
   {{ parameter.name }}:
     type: {{ parameter.type }}
-    description: "{{ parameter.description }}"
```

Comment:
Just curious why you changed that?

It looks like previously the script replaced `"` with `'`.